test/meson.build

```diff
@@ -0,0 +1,76 @@
+add_languages('cpp')
+
+gtest_sources = [
+  'gtest/src/gtest-all.cc',
+  'gtest/src/gtest_main.cc',
+]
+
+gtest_includes = [
+  include_directories('gtest'),
+  include_directories('gtest/include'),
+]
+
+libgtest = static_library(
+  'gtest',
+  gtest_sources,
+  include_directories : gtest_includes)
```

Comment:
`dependencies : thread_dep`?  Autotools has it implicit in AM_LDFLAGS.

c++11 std? Autotools has it implicit in AM_CPPFLAGS

Reply:
> c++11 std? Autotools has it implicit in AM_CPPFLAGS

In test's meson.build:

```
  include_directories : gtest_includes,
  override_options : [ 'cpp_std=c++11' ])
```

Reply:
Yes, but it is only specified in the test_i965 target and not the libgtest target.  It needs to be in both targets.

Reply:
> Yes, but it is only specified in the test_i965 target and not the libgtest target. It needs to be in both targets.

Indeed.